cookbook/integrations/aws/sagemaker_pytorch/Dockerfile

```diff
@@ -7,6 +7,11 @@ WORKDIR /root
 ENV LANG C.UTF-8
 ENV LC_ALL C.UTF-8
 ENV PYTHONPATH /root
+ENV DEBIAN_FRONTEND noninteractive
+ENV TERM linux
+
+# Install basics
+RUN apt-get update && apt-get install -y make build-essential libssl-dev curl
```

Comment:
is this needed?

Reply:
we install make couple lines below `apt-get install make` we can drop that maybe?

Reply:
ssl is what it was complaining about... we might be able to drop the others...